dev/statement_serializer.h

```diff
@@ -1091,6 +1091,22 @@ namespace sqlite_orm {
             }
         };
 
+        template<class T>
+        struct statement_serializer<table_content_t<T>, void> {
+            using statement_type = table_content_t<T>;
+
+            template<class Ctx>
+            std::string operator()(const statement_type& statement, const Ctx& context) const {
+                using mapped_type = typename statement_type::mapped_type;
+
+                auto& table = pick_table<T>(context.db_objects);
```

Comment:
Use `mapped_type` instead of `T` for the sake of completeness?

Reply:
done